data-description/fair-data.md

```diff
@@ -0,0 +1,128 @@
+---
+title: FAIR data
+description: Apply the FAIR principles to infectious disease research data, emphasizing documentation, accessibility, metadata standards, and interoperability to promote transparency, collaboration, and reuse.
+contributors: ["Marcos Casado Barbero", "Romain David", "Iris Van Dam", "Rudolf Wittner"]
+page_id: fd_data_description
+redirect_from: /fair-data/data-description
+rdmkit:
+  - name: Why are the FAIR principles needed?
+    url: https://rdmkit.elixir-europe.org/about#why-are-the-fair-principles-needed
+  - name: Identifiers
+    url: https://rdmkit.elixir-europe.org/identifiers
+  - name: Sensitive data
+    url: https://rdmkit.elixir-europe.org/sensitive_data
+  - name: What is data reuse
+    url: https://rdmkit.elixir-europe.org/reusing#what-is-data-reuse
+fairsharing:
+  - name: European Genome-phenome Archive (EGA)
+    url: https://fairsharing.org/FAIRsharing.mya1ff
+  - name: European Nucleotide Archive (ENA)
+    url: https://fairsharing.org/FAIRsharing.dj8nt8
+  - name: Database of Genotypes and Phenotypes (dbGaP)
+    url: https://fairsharing.org/FAIRsharing.88v2k0
+  - name: Ontology Lookup Service (OLS)
+    url: https://fairsharing.org/FAIRsharing.Mkl9RR
+related_pages:
+    data_sources: [hchd_data_sources]
+
+# More information on how to fill in this metadata section can be found here https://www.infectious-diseases-toolkit.org/contribute/page-metadata
+---
+
+## Introduction
+
+The [FAIR principles](https://www.go-fair.org/fair-principles/) provide guidelines for making research data (i.e. digital assets) **F**indable, **A**ccessible, **I**nteroperable, and **R**eusable ([Wilkinson et al., 2016](https://doi.org/10.1038/sdata.2016.18 )). In infectious diseases research, adhering to these principles is crucial for facilitating data sharing, collaboration, and accelerating progress towards:
+* Meta-analyses
+* Reliable transmission maps
+* Understanding of the spread and evolution of pathogens
+* Improved diagnostics, treatments, and vaccines
+
+By ensuring maximal data usability, FAIRness increases the efficiency and impact of your infectious disease research. 
+
+
+## Findability
+
+Findability is a crucial aspect of infectious diseases research, as it ensures that relevant data and resources can be easily located and accessed by researchers and other stakeholders. 
```

Comment:
Maybe easily **discovered and located** instead of **located and accessed** as this already talks about Accessibility.